.gitmodules

```diff
@@ -4,3 +4,8 @@
 [submodule "Yoga"]
 	path = current/Yoga
 	url = https://github.com/ReactWindows/yoga
+[submodule "folly"]
+	path = GitModules/folly
+	url = https://github.com/jurocha-ms/folly.git
```

Comment:
>https://github.com/jurocha-ms/folly.git [](start = 7, length = 39)

Why is this in your personal repo?

Reply:
We need to fork to apply our patches, and we don't have a `microsoft/folly` repository.
This can be changed and fixed in the future trivially.
Moreover, if Facebook accepts this change, we can go back to referencing their upstream repo.